src/components/FeatureFlagModal/FeatureFlagModal.tsx

```diff
@@ -247,6 +248,14 @@ export default function FeatureFlagModal() {
           featureFlag={FeatureFlag.fotAdjustedmentsEnabled}
           label="Enable fee-on-transfer UI and slippage adjustments"
         />
+        <FeatureFlagGroup name="Quick routes">
```

Comment:
going to use a statsig dynamic config for enabling the other chains instead of a single feature flag, since support for the other chains will be on a slow rollout